app/controllers/doorkeeper/authorizations_controller.rb

```diff
@@ -41,9 +41,9 @@ def redirect_or_render(auth)
     end
 
     def pre_auth
-      @pre_auth ||= OAuth::PreAuthorization.new(Doorkeeper.configuration,
-                                                server.client_via_uid,
-                                                params)
+      @pre_auth ||= Doorkeeper.configuration.preauth_class.new(Doorkeeper.configuration,
```

Comment:
Breaking the line is no problem.  For context, there's similar code in [the use of the access_token_generator option](https://github.com/doorkeeper-gem/doorkeeper/blob/master/lib/doorkeeper/models/access_token_mixin.rb#L132).